scripts/pegin.sh

```diff
@@ -1,33 +1,51 @@
 #!/usr/bin/env bash
+# Calls the CLI to peg into the federation
 
 set -e
 
 # Let's define some shortcuts for bitcoind and the mint client
-BTC_CLI="bitcoin-cli -regtest -rpcconnect=127.0.0.1 -rpcuser=bitcoin -rpcpassword=bitcoin"
-CLIENT="cargo run --release --bin mint-client-cli cfg"
+POLL_INTERVAL=1
+CONFIRMATION_TIME=10
```

Comment:
I think we should read that from the config (currently server config, after #216 client config) or even the federation itself #218

Reply:
I'll do that, although it's a bit hacky and should be replaced with a proper API at some point.